pkg/nodeconfig/nodeconfig.go

```diff
@@ -149,6 +149,10 @@ func (nc *nodeConfig) Configure() error {
 		return err
 	}
 
+	if err := nc.Windows.DeconfigureWICD(); err != nil {
+		return errors.Wrap(err, "deconfiguring WICD failed")
```

Comment:
Why does this need to be done here? Please add a comment

Reply:
Additionally it should be done before createBootstrapFiles if it is necessary, it is odd to start bootstrapping and then clean something up in the middle.

Reply:
Please see this [conversation](https://github.com/openshift/windows-machine-config-operator/pull/1332/commits/0486bd02e166e0319d35cf11ed501849851c2b8c#r1038511228). I can move the call site.

Reply:
That link doesn't work for me, but regardless a comment on the PR doesn't help someone reading the code, please add a comment.